presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -83,6 +85,9 @@
 {
     private static final Logger log = Logger.get("TestQueries");
 
+    private static final DateTimeFormatter timeWithUtcZoneFormat = DateTimeFormatter.ofPattern("HH:mm:ss.SSS 'UTC'"); // UTC zone would be printed as "Z" in "XXX" format
```

Comment:
> // UTC zone would be printed as "Z" in "XXX" format

And it prevents the pattern "HH:mm:ss.SSS XXX" from parsing it?

Edit. Oh and you need to handle that differently below.

Question - why do we care about UTC?

Reply:
Because `com.facebook.presto.spi.type.TimeZoneKey#getTimeZoneKey(java.lang.String)` (or, in fact, `com.facebook.presto.spi.type.TimeZoneKey#normalizeZoneId`) and `com.facebook.presto.spi.type.TimeZoneKey#getTimeZoneKeyForOffset` dictates that "UTC" is the normalized ID.